journal.log

```diff
@@ -0,0 +1,4 @@
+5:51 :: ddashenkov :: 2023-06-27T12:00:52.733963Z :: core:2.0.0-SNAPSHOT.141;ProtoData:0.8.5;Validation:2.0.0-SNAPSHOT.81;mc-java:2.0.0-SNAPSHOT.151
+5:47 :: ddashenkov :: 2023-06-27T10:33:46.829737Z :: core:2.0.0-SNAPSHOT.141;ProtoData:0.8.5;Validation:2.0.0-SNAPSHOT.81;mc-java:2.0.0-SNAPSHOT.151
+5:31 :: ddashenkov :: 2023-06-27T10:25:45.749858Z :: core:2.0.0-SNAPSHOT.141;ProtoData:0.8.5;Validation:2.0.0-SNAPSHOT.81;mc-java:2.0.0-SNAPSHOT.151
+3:54 :: ddashenkov :: 2023-06-27T09:26:12.968953Z :: core:2.0.0-SNAPSHOT.141;ProtoData:0.8.5;Validation:2.0.0-SNAPSHOT.81;mc-java:2.0.0-SNAPSHOT.151
```

Comment:
I think this file should not be a part of this PR. Not sure though, but I don't get the idea of including it.

Reply:
I intend on keeping this file in the repo, so that the build journals in other projects are also kept in the Git repos.

I've added a note on this in the README.